### ISSUE TITLE
Cant save a facebook webpage offline as html, the three "save as" functions dont work.

### ISSUE THREAD

**Issue by Username23453** [Alex313031/thorium#663]
I'm using ZorinOS, am using current release of Thorium.

I have to install this extension, in order to save a webpage offline as .mhtml.
https://chromewebstore.google.com/detail/save-webpages-offline-as/nfbcfginnecenjncdjhaminfcienmehn

**Comment by Alex313031** [comment]
@Username23453 I don't think this is Thorium-specific. I can't save Facebook pages normally either in Chromium.
